src/clue/clue.sass

```diff
@@ -37,6 +37,17 @@
     background-color: $sky-dark-2
     border-color: $sky-light-3
 
+  .support-badge
+    $badge-radius: 9px
+    position: absolute
+    left: -$padding - ($badge-radius / 4)
+    top: 0
+    background-color: #fff
+    border: 4px solid $sky
```

Comment:
I'm not seeing any handling of the `:hover` state here. In the prototype it switches to blue with a white outline on hover.

Reply:
perhaps, although I'm not all that concerned about that. Only that it should disappear once the tab is opened.